forms/OptionsetField.php

```diff
@@ -90,31 +88,15 @@ public function Field($properties = array()) {
 		);
 	}
 
-	public function performReadonlyTransformation() {
```

Comment:
Second commit - it’s an exact duplicate of `DropdownField:: performReadonlyTransformation()`